docs: useUserOrderByCart docs update (#358)
vuestorefront/bigcommerce: docs/composables/use-user-order-by-cart.md

## Features

> `useUserOrderByCart` composable is used to get shipping information for fulfilled and active user orders related to the given cart. 

Reviewer: ```suggestion
The `useUserOrderByCart` composable is used to get shipping information for fulfilled and active user orders related to the given cart. 
```

## API

> - `load` - the main querying function used to query the user order by cart id from the BigCommerce platform and populate the `order` object with the result. It accepts a single object as a parameter. See the [CategorySearchParams](../reference/api/bigcommerce.categorysearchparams.md) interface for more information.

Reviewer: ```suggestion
- `load` - a function for querying the user order by cart ID from the API and populating the `order` object with the returned data. It accepts a single object as a parameter. See the [CategorySearchParams](../reference/api/bigcommerce.categorysearchparams.md) interface for more information.
```

- `order: OrderByCartResponse` - the main data object.

> - `loading: boolean` - a reactive object containing information about the loading state of your `load` method.

Reviewer: ```suggestion
- `loading: boolean` - a reactive object containing information about the loading state of the `load` method.
```

> - `error: UseUserOrderByCartErrors` - reactive object containing the error message, if `load` failed for any reason. See the [UseUserOrderByCartErrors](../reference/api/bigcommerce.useuserorderbycarterrors.md) interface for more information.

Reviewer: ```suggestion
- `error: UseUserOrderByCartErrors` - a reactive object containing the error message, if the `load` failed for any reason. See the [UseUserOrderByCartErrors](../reference/api/bigcommerce.useuserorderbycarterrors.md) interface for more information.
```
